src/management/Akka.Management/Cluster/Bootstrap/Internal/HttpContactPointBootstrap.cs

```diff
@@ -94,6 +95,8 @@ public HttpContactPointBootstrap(ClusterBootstrapSettings settings, ResolvedTarg
             _probeInterval = settings.ContactPoint.ProbeInterval;
             _probeRequest = ClusterBootstrapRequests.BootstrapSeedNodes(baseUri);
             _cancellationTokenSource = new CancellationTokenSource();
+
+            _probingFailureTimeout = _probeInterval + _settings.ContactPoint.ProbingFailureTimeout;
```

Comment:
Probe timeout value is modified to take consideration the actual probing interval time span